include/sharpSAT/primitive_types.h

```diff
@@ -19,22 +17,13 @@ typedef unsigned ClauseOfs;
 typedef unsigned CacheEntryID;
 
 static const ClauseIndex NOT_A_CLAUSE(0);
-#define SENTINEL_CL NOT_A_CLAUSE
-
+static const auto SENTINEL_CL = NOT_A_CLAUSE;
+static const auto clsSENTINEL = NOT_A_CLAUSE;
```

Comment:
Do these two need to be split? I did quick search over the code base and they seem to be used interchangeably in most places.